### ISSUE TITLE
Error with Pyinstrument 4.0.0: RuntimeError: Could not find app.js. If you are running pyinstrument from a git checkout, run 'python setup.py build' to compile the Javascript (requires nodejs). 

### ISSUE THREAD

**Issue by ndodda-amazon** [joerick/pyinstrument#137]
Python Version: 3.7.10

OS: Ubuntu 18.04.5

Steps to reproduce:

```
pip install pyinstrument==4.0.0
python test.py
```

where `test.py` is:

```
import time
from pyinstrument import Profiler

profiler = Profiler()

profiler.start()

time.sleep(1)

profiler.stop()

profiler.output_html()
```

And the full stack trace of the error is:

```
Traceback (most recent call last):
  File "test.py", line 14, in <module>
    profiler.output_html()
  File "/usr/local/lib/python3.7/site-packages/pyinstrument/profiler.py", line 295, in output_html
    return renderers.HTMLRenderer(timeline=timeline).render(self.last_session)
  File "/usr/local/lib/python3.7/site-packages/pyinstrument/renderers/html.py", line 26, in render
    "Could not find app.js. If you are running "
RuntimeError: Could not find app.js. If you are running pyinstrument from a git checkout, run 'python setup.py build' to compile the Javascript (requires nodejs).
```

Additional Notes:
The error seems to imply that I'm building pyinstrument from source, but I'm installing from PyPi. The error does not occur with pyinstrument 3.4.2 or an earlier version. 

Oddly enough, I'm only able to get this error on a Linux machine. On my laptop (Unix) I don't get the error.

Please let me know if you need any additional information to debug this error.

**Comment by joerick** [comment]
thanks for the report. looking into it!

**Comment by joerick** [comment]
the JS files were indeed missing - I've yanked that release and created 4.0.1 that has only a source distribution for now... next, I'll try to figure out why they were missing!

**Comment by joerick** [comment]
Haha. The lamest bug. "Manifest.in" != "MANIFEST.in" on linux! But it does equal that on mac/windows.

**Comment by joerick** [comment]
Fixed in v4.0.2. if you could confirm your end, that would be great :)

**Comment by ndodda-amazon** [comment]
Thanks @joerick, this issue was resolved but now I'm seeing a different issue with v4.0.2. I filed it here: https://github.com/joerick/pyinstrument/issues/139. Can you take a look?
